src/styled-components.d.ts

```diff
@@ -17,7 +17,7 @@ declare module "styled-components" {
   };
 
   export interface DefaultTheme {
-    grey: Omit<ColorRange, "-3" | "-2" | "+3">;
+    grey: Omit<ColorRange, "-3" | "-2">;
```

Comment:
Looks like the types were wrong prior to this PR - each key in the theme.ts should be reflected in here

```suggestion
    grey: Omit<ColorRange, "-3" | "-2" | "-1" | "0" | "+1" | "+2" | "+3">;
```

Reply:
Do you think the types should be this strict? Part of me things that all colour options should allow a value between `-3` and `+3`